src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #00acdc;
+  --background-color: #fff;
+  --text-color: #000;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  font-style: normal;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: var(--background-color);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

Comment:
1. instead of rewriting your margin only for last-child you can use `:not(:first-child)` and set margin only for not last elements
2. i think you don't need position relative here, it's not used